package/MDAnalysis/coordinates/XYZ.py

```diff
@@ -161,6 +168,8 @@ def __init__(self, filename, n_atoms=None, convert_units=True,
         remark: str (optional)
             single line of text ("molecule name"). By default writes MDAnalysis
             version and frame
+        precision: int (optional)
+            set precision of saved trjactory to this number of decimal places.
```

Comment:
add a `.. versionadded:: 2.9.0` for the new kwarg
```suggestion
            set precision of saved trjactory to this number of decimal places.

            .. versionadded:: 2.9.0
```